app/utils/utils.ts

```diff
@@ -45,6 +45,7 @@ export const getItemsCountFromUUIDs = async (uuids: string[]) => {
   if (!response?.counts?.count?.length) {
     return {};
   }
+  console.log("helper response", response);
```

Comment:
I have a feeling the `return` is hit before so it might not log this at all. Move this up before line 45.